users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -811,6 +811,15 @@ def run_scf_specaug():
 
     nn_args, report_args_collection = get_nn_args_baseline(
         nn_base_args={
+            "old_baseline": dict(
```

Comment:
```suggestion
            "baseline": dict(
```
Please call the existing thing is the baseline, not the new stuff.